source/slang/slang-capabilities.capdef

```diff
@@ -65,7 +68,22 @@ alias any_target = hlsl | glsl | c | cpp | cuda | spirv;
 alias any_textual_target = hlsl | glsl | c | cpp | cuda;
 alias any_gfx_target = hlsl | glsl | spirv;
 alias any_cpp_target = cpp | cuda;
+alias cpp_cuda = cpp | cuda;
+alias cpp_glsl_cuda_spirv = cpp | glsl | cuda | spirv;
+alias cpp_hlsl = cpp | hlsl;
+alias cpp_hlsl_cuda = cpp | hlsl | cuda;
+alias cpp_hlsl_glsl_cuda = cpp | hlsl | glsl | cuda;
+alias cpp_hlsl_glsl_cuda_spirv = cpp | hlsl | glsl | cuda | spirv_1_0;
+alias cpp_hlsl_glsl_spirv = cpp | hlsl | glsl | spirv_1_0;
+alias hlsl_cuda = hlsl | cuda;
+alias hlsl_cuda_spirv = hlsl | cuda | spirv;
+alias hlsl_glsl_cuda = hlsl | glsl | cuda;
+alias hlsl_glsl_spirv = hlsl | glsl | spirv;
+alias hlsl_glsl_cuda_spirv = hlsl | glsl | cuda | spirv_1_0;
+alias hlsl_spirv = hlsl | spirv;
+alias glsl_cuda_spirv = glsl | cuda | spirv;
 alias glsl_spirv = glsl | spirv;
+
```

Comment:
What is the rule of the ordering the keywords?
Can we sort them by alphabet order like this?
```
alias cpp_cuda = cpp | cuda;
alias cpp_cuda_glsl_hlsl = cpp | cuda | glsl | hlsl;
alias cpp_cuda_glsl_hlsl_spirv = cpp | cuda | glsl | hlsl | spirv_1_0;
alias cpp_cuda_glsl_spirv = cpp | cuda | glsl | spirv;
alias cpp_cuda_hlsl = cpp | cuda | hlsl;
alias cpp_glsl_hlsl_spirv = cpp | glsl | hlsl | spirv_1_0;
alias cpp_hlsl = cpp | hlsl;
alias cuda_glsl_hlsl = cuda | glsl | hlsl;
alias cuda_glsl_hlsl_spirv = cuda | glsl | hlsl | spirv_1_0;
alias cuda_glsl_spirv = cuda | glsl | spirv;
alias cuda_hlsl = cuda | hlsl;
alias cuda_hlsl_spirv = cuda | hlsl | spirv;
alias glsl_hlsl_spirv = glsl | hlsl | spirv;
alias glsl_spirv = glsl | spirv;
alias hlsl_spirv = hlsl | spirv;
```

Reply:
The rule was [term1]_[term2]... where [term1] and length have priority over term2 for ordering.

I can sort alphabetically.